### ISSUE TITLE
license gen error while running run.sh interactive

### ISSUE THREAD

**Issue by robina80** [jakeswenson/BitBetter#35]
hi all,

im getting the below error when im running the run script to produce a license - 

root@bitwarden_ubuntu:/bitwarden/bwdata/BitBetter/src/licenseGen# /bitwarden/bwdata/BitBetter/src/licenseGen/run.sh interactive
Cant find certificate at: /cert.pfx

Usage:  interactive

any help would be much appreciated

thanks,
rob

**Comment by robina80** [comment]
yes...

stupid me it was in the how to - 

root@bitwarden_ubuntu:/bitwarden/bwdata/BitBetter/src/licenseGen# ./run.sh /bitwarden/bwdata/BitBetter/.keys/cert.pfx interactive

it generated it on screen and then i copy/pasted the text in a file and called it blah.json

**Comment by DB-CL** [comment]
Hmm, still, I have the same issue when I run interactive from the main folder :( I had to specify the cert.pfx path even though it's located in the default folder.  Not a big deal but not very smooth neither.

**Comment by JeffAlyanak** [comment]
@DB-CL The `./src/licneseGen/run.sh` script was looking in the wrong place for the key directory.

This is fixed in 8f3fc3014d45be89193768b2c442a9a12a044ab5, I've put in a pull request for this change but, as you can see, it's a very simple change that you can even do yourself.

Simply replacing:
```bash
cert_path="$DIR/.keys/cert.pfx"
```
With:
```bash
cert_path="$DIR/../../.keys/cert.pfx"
```

**Comment by JeffAlyanak** [comment]
@DB-CL 

Fix has to merged to the master branch.
